spinn_machine/multicast_routing_entry.py

```diff
@@ -154,20 +168,30 @@ def __or__(self, other_entry):
     def __eq__(self, other_entry):
         if not isinstance(other_entry, MulticastRoutingEntry):
             return False
-        return (self._defaultable == other_entry.defaultable and
-                self._link_ids == other_entry.link_ids and
-                self._mask == other_entry.mask and
-                self._processor_ids == other_entry.processor_ids and
-                self._routing_entry_key == other_entry.routing_entry_key)
+        if self.routing_entry_key != other_entry.routing_entry_key:
+            return False
+        if self.mask != other_entry.mask:
+            return False
+        if self._spinnaker_route != other_entry._spinnaker_route:
+            return False
+        if self._link_ids != other_entry._link_ids:
```

Comment:
Is it necessary to check link_ids?  Note that if the spinnaker_route *does* match, this check will be done, and I would expect it to be True as well...

Reply:
No it is not necessary to check link ids or processor ids.

In fact in later branches this caused trouble when sometimes list where sued and sometimes sets.

Left in as I did not want to take out a check others where used to.